command_line/find_spots.py

```diff
@@ -177,6 +197,14 @@ def run(self, args=None):
 
         # Save the experiments
         if params.output.experiments:
+
+            # Reset the trusted ranges
+            if params.maximum_trusted_value > 0:
+                for detector in experiments.detectors():
+                    for panel in detector:
+                        trusted = input_trusted_ranges[id(panel)] = trusted
+                        panel.set_trusted_range(trusted)
+
```

Comment:
I think you've left a bit of copypasta in (`trusted = input_trusted_ranges[id(panel)] = trusted`).  Also, I think we probably want to reset this regardless of the value of `params.output.experiments` in case we want to do anything afterwards with the experiment list before the process finishes (for example it might have a bearing on the later `params.per_image_statistics` block.

```suggestion

        # Reset the trusted ranges
        if params.maximum_trusted_value:
            for detector in experiments.detectors():
                for panel in detector:
                    trusted = input_trusted_ranges[id(panel)]
                    panel.set_trusted_range(trusted)

        # Save the experiments
        if params.output.experiments:

```

Reply:
Good suggestion, implemented

Reply:
I note that you removed the copypasta but didn't implement the logic change.  Any reason for that?

Reply:
Nor did you remove the second redundant `> 0` as suggested here.